### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by prithviBytes**
HI Team,
I have completed the project. Please review it.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,3 @@
-# demo_pr
\ No newline at end of file
+# demo_pr
+
+COmpleted project.
```
Comment (themohammadsa) [inline README.md:3]: Please check the use-case.

**Review by themohammadsa — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (prithviBytes) [reply README.md]: Hi,
Please check the changes.

**Review by themohammadsa — APPROVED** [review]
Approved.

**Review by themohammadsa — COMMENTED** [review]
test

**Pull request merged**